spec/features/filter_spec.rb

```diff
@@ -47,5 +47,16 @@
         end
       end
     end
+
+    context 'when on the homepage' do
+      before do
+        visit '/'
+      end
+
+      it 'filters by organisation' do
+        find('a[data-cy="Test Organisation-filter-link"]').click
+        expect(page).to have_content 'Test Data Service'
```

Comment:
Looks like this test would pass even if the filter wasn't being applied, since the default is to show all the services. Perhaps this should instead check the number of services on the page?